modules/get-started/pages/licensing/overview.adoc

```diff
@@ -31,27 +31,80 @@ The Enterprise Edition is licensed with the https://github.com/redpanda-data/red
 
 === License keys
 
-Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license. You can evaluate enterprise features with a free 30-day trial. To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
+Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license.
+
+Starting with version 24.3, new Redpanda clusters automatically receive a trial license that's valid for 30 days, allowing unrestricted use of Enterprise features. This evaluation period begins when the cluster is created for the first time. After this period expires, inactive Enterprise features are disabled, and active features enter a restricted state.
```

Comment:
Should we expand on what is a restricted state?

Reply:
It's expanded on in the table. Restricted state means different things depending on the feature.